sdk/metricsadvisor/tests.yml

```diff
@@ -10,7 +10,7 @@ extends:
       METRICSADVISOR_PRIMARY_API_KEY: $(net-metricsadvisor-test-api-key)
       METRICSADVISOR_DATA_FEED_ID: "9860df01-e740-40ec-94a2-6351813552ba"
       METRICSADVISOR_METRIC_ID: "27e3015f-04fd-44ba-a20b-bc529a0aebae"
-      METRICSADVISOR_DETECTION_CONFIGURATION_ID: "fb5a6ed6-2b9e-4b72-8b0c-0046ead1c15c"
+      METRICSADVISOR_DETECTION_CONFIGURATION_ID: "35c5ee94-8512-4cdc-afc0-fdbb36e38569"
```

Comment:
We had a "resource leakage" some time ago in our tests (already fixed) and ended up reaching the quota for detection configurations, so I did some cleaning up. I created a new detection configuration with a very clear name (NET_SAMPLES_DONT_DELETE) so no one else ends up deleting it.

Reply:
Ok. This one is pretty messy. A lot of things depend on the Detection Configuration ID. I'll create another PR to fix this before merging this one.